src/schema/v2/homeView/__tests__/HomeView.test.ts

```diff
@@ -290,14 +306,6 @@ describe("homeView", () => {
                   },
                 },
               },
-              Object {
-                "node": Object {
-                  "__typename": "HomeViewSectionShows",
-                  "component": Object {
-                    "title": "Shows for You",
-                  },
-                },
-              },
```

Comment:
I think we didn't mean to drop this from the snapshot. This likely resulted from the hard-coded size on L16 above. 

**suggestion:** Would you kindly bump that up, e.g. `20` → `30`?

Reply:
oh mb, I think I've updated the snapshot directly and forget to check the diff 😓 
if I do it I see another node that wasn't there before, is it fine if I add it too?

```
    +     Object {
    +       "node": Object {
    +         "__typename": "HomeViewSectionShows",
    +         "component": Object {
    +           "title": "Shows for You",
    +         },
    +       },
    +     },
    +     Object {
    +       "node": Object {
    +         "__typename": "HomeViewSectionFairs",
    +         "component": Object {
    +           "title": "Featured Fairs",
    +         },
    +       },
    +     },
    
```

Reply:
Yes, please! That one should be there too